internal/ovirt/resource_ovirt_vm.go

```diff
@@ -164,6 +164,20 @@ var vmSchema = map[string]*schema.Schema{
 		Optional:    true,
 		Description: "Enable or disable the serial console.",
 	},
+	"clone": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Description: "If true, the VM is cloned from the template instead of linked. As a result, the template can be removed and the VM still exists.",
+	},
+}
+
+func cpuModeValues() []string {
```

Comment:
Why is this code in this PR?

Reply:
Just moved this function up so all `xyzValues()` are close together.